deps/g3dlite/include/G3D/System.h

```diff
@@ -20,7 +20,7 @@
 #include "G3D/BinaryFormat.h"
 #include "G3D/FileNotFound.h"
 #include <string>
-
+#include <sys/time.h>
 #if defined(__aarch64__)
 #include <sys/time.h>
```

Comment:
Breaks windows, thats why the if aarch64 is there.

Reply:
It fails to build on fbsd if its done within the if, hm

Reply:
Include if FreeBDS?

Reply:
```cpp
# if defined(__aarch64__) || defined(__FreeBSD__)
```